index.html

```diff
@@ -1,11 +1,43 @@
 <html>
-
+ 
 <head>
-	<meta charset="utf-8">
-	<meta name="viewport" content="width=device-width">
-	<title>repl.it</title>
-	<link href="main.css" rel="stylesheet" type="text/css" />
-	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css">
-	<script src="main.js">
-	</script>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>repl.it</title>
+  <link href="main.css" rel="stylesheet" type="text/css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css">  
+  <script src="main.js">
+
+
+  </script 
 </head>
+
+<body>
+
+  <div class="container">
+    <div class="answer"> </div>
+    <div class="box1"> C </div>
+    <div class="box1">7</div>
+      <div class="box1">8</div>
+  <div class="box1">9</div>     
+  <div class="box1">X</div>
+      <div class="box1">4</div>
+  <div class="box1">5</div>
+  <div class="box1">6</div>
+      <div class="box1">/</div>
+  <div class="box1">1</div>
+  <div class="box1">2</div>
+      <div class="box1">3</div>
+  <div class="box1">-</div>
+  <div class="box1">.</div>
+      <div class="box1">0</div>
+  <div class="box1">Enter</div>
+  <div class="box1">+</div>
+
+
+
+
+
+
+  </div>
+</body>   
```

Comment:
I see the progress boss! I wanna see how you implement the JavaScript code we were looking at Zoom to get the button workings. Love your persistence